### PR TITLE
Update RELEASE.md

### PR DESCRIPTION
**Pull request by hilcj**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -186,6 +186,8 @@ Note, when releasing from master, all the below mentions of "release branch" mea
     ./hack/release.sh $VERSION $BRANCH
     ```
     It will prompt you whether to push it to release branch. Press `y` and hit `Enter`.
+    
+    First time running this command may need to setup the PATH environmental variable to include XXX/go/bin.  Potential location of the XXX/go/bin are /usr/local/go/bin, ~/go/bin.  Once found, run ```export PATH=$PATH:~/go/bin``` before running ```release.sh```.
```
Comment (Bobgy) [inline RELEASE.md:190]: I think we should better link to go official documentation about why we need to do this.

Something I found:
```
$ go env
```
returns current env vars for golang.

`$GOPATH/bin` is used to store fetched binaries and default value for `$GOPATH` is `~/go`.

https://golang.org/cmd/go/#hdr-GOPATH_and_Modules

**Comment by Bobgy** [comment]
Another thing, is that we should follow title convention for PRs (they were auto-populated in PR description when you create one).

**Comment by Bobgy** [comment]
Thanks for keeping the doc up-to-date!

**Comment by hilcj** [comment]
Merged with PR #4832

**Pull request closed without merging**